modules/stochastic_tools/examples/surrogates/cross_validation/#uniform_train_and_cv.i#

```diff
@@ -0,0 +1,101 @@
+[StochasticTools]
```

Comment:
This looks like an autosave file. Meant to be checked in?

Reply:
Yeah, that shouldn't have been there. I'll be better about checking for these from now on.  Removed it